android/src/main/java/com/evollu/react/fcm/FIRMessagingModule.java

```diff
@@ -54,12 +60,38 @@ public void requestPermissions(){
     }
 
     @ReactMethod
+    public void getInitFCMData(Promise promise) {
```

Comment:
Trying to get the data payload that is delivered in the extras of the intent after opening the app from a notification in the system tray. As documented [here](https://firebase.google.com/docs/cloud-messaging/android/receive)